src/Google/Ads/GoogleAds/Util/V1/ResourceNames.php

```diff
@@ -162,88 +504,1006 @@ public static function forCampaignCriterion($customerId, $campaignId, $criterion
+     *
+     * @param int $mobileAppCategoryId the mobile app category ID
+     * @return string the mobile app category constant resource name
+     */
+    public static function forMobileAppCategoryConstant($mobileAppCategoryId)
+    {
+        return MobileAppCategoryConstantServiceClient::mobileAppCategoryConstantName(
+            $mobileAppCategoryId
+        );
+    }
+
+    /**
+     * Generates resource name for a mobile device constant.
+     *
+     * @param int $criterionId the criterion ID
+     * @return string the mobile device constant resource name
+     */
+    public static function forMobileDeviceConstant($criterionId)
+    {
+        return MobileDeviceConstantServiceClient::mobileDeviceConstantName($criterionId);
+    }
+
+    /**
+     * Generates resource name for a mutate job.
+     *
+     * @param int $customerId the customer ID
+     * @param int $mutateJobId the mutate job ID
+     * @return string the mutate job resource name
+     */
+    public static function forMutateJob($customerId, $mutateJobId)
+    {
+        return MutateJobServiceClient::mutateJobName(
+            $customerId,
+            $mutateJobId
+        );
+    }
+
+    /**
+     * Generates resource name for a operating system version constant.
+     *
+     * @param int $criterionId the criterion ID
+     * @return string the operating system version constant resource name
+     */
+    public static function forOperatingSystemVersionConstant($criterionId)
+    {
+        return OperatingSystemVersionConstantServiceClient::operatingSystemVersionConstantName(
+            $criterionId
+        );
+    }
+
+    /**
+     * Generates resource name for a paid organic search term view.
+     *
+     * @param int $customerId the customer ID
+     * @param int $campaignId the campaign ID
+     * @param int $adGroupId the ad group ID
+     * @param string $searchTerm the search term
+     * @return string the paid organic search term view resource name
+     */
+    public static function forPaidOrganicSearchTermView(
+        $customerId,
+        $campaignId,
+        $adGroupId,
+        $searchTerm
+    ) {
+        return PaidOrganicSearchTermViewServiceClient::paidOrganicSearchTermViewName(
+            $customerId,
+            "{$campaignId}~{$adGroupId}~{$searchTerm}"
+        );
+    }
+
+    /**
+     * Generates resource name for a parental status view.
+     *
+     * @param int $customerId the customer ID
+     * @param int $adGroupId the ad group ID
+     * @param int $criterionId the criterion ID
+     * @return string the parental status view resource name
+     */
+    public static function forParentalStatusView($customerId, $adGroupId, $criterionId)
+    {
+        return ParentalStatusViewServiceClient::parentalStatusViewName(
+            $customerId,
+            "{$adGroupId}~{$criterionId}"
+        );
+    }
+
+    /**
+     * Generates resource name for a paid organic search term view.
```

Comment:
Please fix wrong PHPDoc here and in @return

Reply:
Done